packages/coil-client/src/graphQlClient.ts

```diff
@@ -48,23 +52,57 @@ export class GraphQlClient {
     @inject(GraphQlClientOptions)
     private config: GraphQlClientOptions
   ) {
-    this.fetch = this.config.fetch
+    this.inBrowser = typeof window !== 'undefined'
+    this.fetch = this.inBrowser
```

Comment:
where had this been a problem?

Reply:
I'll see if I can try to reproduce the issue, but my browser throws an error if you attempt to call `fetch` unbounded to `window`.

```js
class FetchTest {
    constructor() {
        this.fetch = window.fetch
    }
}
const f = new FetchTest()
f.fetch('/') // Throws "TypeError: Failed to execute 'fetch' on 'Window': Illegal invocation"
```

Reply:
The portableFetch we use everywhere is a wrapper function (wrapper = `(...args) => fetch(...args)` essentially) so this shouldn't be needed. There was a time in main relatively recently where there was a regression circa this though. Maybe you copped that.

Reply:
I'll pull that out and see if it's still happening, I bet you're right that I just caught that regression.